### DISCUSSION TITLE
Run copier in task

### DISCUSSION THREAD

**Discussion by tpluscode** [copier-org/copier#639] [Q&A] [answered]
Hi. I. have this app template, which in a task run `copier copy` of a "api" template. Possibly multiple times, based on answers.

Right now that is a `.copier/generate_apis.sh` script, executed in a task

```
tasks:
  - "sh .copier/generate_apis.sh"
```

How would I run copier directly, using python and not default shell in the task?

Alternatively, change `.sh.jinja` into `.py.jinja`. We tried this task:

```
tasks:
  - ["{{ _copier_python }}", '.copier/generate_apis.py']
```

but this way, the output was

```
> Running task 2 of 3: ['',  '.copier/_generate_apis.py']
```

Maybe `_copier_python` is not yet available in `6.0.0a9`?

**Comment by yajo** [comment, marked as answer]
> Maybe `_copier_python` is not yet available in `6.0.0a9`?

Yep. Try v6.0.0b0, released today.

**Reply by tpluscode** [reply]
Awesome. Works like a charm

**Reply by tpluscode** [reply]
A quick follow-up question. Is it possible to reference scripts from the template repo and not the generated repo?

See, in our setup the `.copier` dir is part of the generated output which we then delete. Would be nice if it never made it to the generated dir

**Reply by yajo** [reply]
Yes, see this: https://github.com/Tecnativa/doodba-copier-template/blob/38970f25cbeab810a273d7e5dade36f50c965976/copier.yml#L51
